ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -190,6 +200,143 @@ const triggerSet: TriggerSet<Data> = {
+          12.00: output.thirdBait!(),
+          16.00: output.fourthBait!(),
+        };
+        const dir = data.predationDebuff === 'CF7' ? output.left!() : output.right!();
+        return output.text!({ dir: dir, bait: AetheronecrosisMap[parseFloat(matches.duration)] });
+      },
+      outputStrings: {
+        text: {
+          en: '${dir}, ${bait}',
+        },
+        left: {
+          en: 'Left (Wing Side)',
+        },
+        right: {
+          en: 'Right (Snake Side)',
+        },
+        firstBait: {
+          en: 'First Bait (20s)',
+        },
+        secondBait: {
+          en: 'Second Bait (8s)',
+        },
+        thirdBait: {
+          en: 'Third Bait (12s)',
+        },
+        fourthBait: {
+          en: 'Fourth Bait (16s)',
+        },
+      },
+    },
+    {
+      id: 'P6S Predation In First Bait Reminder',
+      // Using Dual Predation (7878)
+      // Delayed to give roughly same notice interval as other bait reminders
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7878', source: 'Hegemone' }),
+      condition: (data) => data.aetheronecrosisDuration === 20.00,
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 4,
+      infoText: (_data, _matches, output) => output.inFirstBait!(),
+      outputStrings: {
+        inFirstBait: {
+          en: 'In (First Bait)',
+        },
+      },
+    },
+    {
+      id: 'P6S Predation In Bait Reminder',
+      // Using Chelic Predation (787B) and Glossal Predation (787A)
+      // Player could get hit at wrong time and still get this trigger
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: ['787A', '787B'], source: 'Hegemone', capture: false }),
+      durationSeconds: 4,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        data.predationCount = data.predationCount + 1;
+
+        // Map duration to the predation count
+        const AetheronecrosisMap: { [duration: number]: number } = {
+          8.00: 1,
+          12.00: 2,
+          16.00: 3,
+        };
+
+        // Output for in players
+        if (AetheronecrosisMap[data.aetheronecrosisDuration] === data.predationCount) {
+          const inBaitMap: { [duration: number]: string } = {
+            1: output.inSecondBait!(),
+            2: output.inThirdBait!(),
+            3: output.inFourthBait!(),
+          };
+          return inBaitMap[data.predationCount];
+        }
+      },
+      outputStrings: {
+        inSecondBait: {
+          en: 'In (Second Bait)',
+        },
+        inThirdBait: {
+          en: 'In (Third Bait)',
+        },
+        inFourthBait: {
+          en: 'In (Fourth Bait)',
+        },
+      },
+    },
+    {
+      id: 'P6S Predation Out',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: ['787A', '787B'], source: 'Hegemone' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.out!(),
```

Comment:
```suggestion
      infoText: (_data, _matches, output) => output.out!(),
```

Maybe infoText to go along with "move" and to not have everything be alerts back to back here.